core_fw/remote_radio/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(remote_radio)
```

Comment:
When it comes to the application name, I agree with @grochu it would be clearer if it indicated `IPC` if I understood the intended scope of this app correctly.